src/Skia/Avalonia.Skia/DrawingContextImpl.cs

```diff
@@ -693,9 +702,14 @@ private void ConfigureGradientBrush(ref PaintWrapper paintWrapper, Size targetSi
                         }
                         else
                         {
+                                
+                            var transformOrigin = radialGradient.TransformOrigin.ToPixels(targetSize);
```

Comment:
We probably can avoid some calculations, if TransformOrigin has default value?

Reply:
Probably, but `RenderTransform` doesn't do a check (I copied this code from there) and I don't think this would cause a huge perf hiccup. Nevertheless, if this feels necessary I'll add it.